app/controllers/admin/exams_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::ExamsController < AdminController
+  def index
+    size = load_per_page Settings.exams.exams_per_page_5
+    result = Exam.by_key_word_with_relation_tables query = params[:query]
```

Comment:
```suggestion
    result = Exam.by_key_word_with_relation_tables params[:query]
```
mi rảnh ghê cu
ah mi học jav... 
cái này cần j tạo biến

Reply:
copy xuống mà bị sót chị..